includes/class-sensei-lesson.php

```diff
@@ -1011,30 +1021,41 @@ public function quiz_update( $post_id ) {
 				}
 			}
 
-			// Set the post terms for quiz-type
+			// Set the post terms for quiz-type.
 			wp_set_post_terms( $quiz_id, array( 'multiple-choice' ), 'quiz-type' );
+
+			/**
+			 * Fires after a quiz is created while saving a lesson in a non-block editor.
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @hook sensei_lesson_quiz_created
+			 *
+			 * @param {int} $quiz_id   Quiz post ID.
+			 * @param {int} $lesson_id Course post ID.
+			 */
+			do_action( 'sensei_lesson_quiz_created', $quiz_id, $post_id );
```

Comment:
Here is the only meaningful change in the file, the rest is the code style fix.

Here we fire an action on the creation of a new quiz within a non-block editor.